docs/integrate/testing.md

```diff
@@ -0,0 +1,56 @@
+(testing)=
+# Software Testing with CrateDB
+
+This documentation section lists Java- and Python-based test frameworks
+and libraries that support software integration testing with CrateDB.
+
+
+(python-pytest)=
+## Python pytest
+
+The popular [pytest] framework makes it easy to write small tests, yet
+scales to support complex functional testing for applications and libraries.
+The [pytest-crate] package manages CrateDB instances for running integration
+tests against them. It is based on [cr8](#cr8), and provides the `crate`,
+`crate_execute`, and `crate_cursor` pytest fixtures on top.
+
+- [Using "pytest-crate" with CrateDB and pytest]
+
+
+(cr8)=
+(python-unittest)=
+## Python unittest
+
+[cr8], a collection of tools for CrateDB developers, provides primitives to
```

Comment:
```suggestion
[cr8], a collection of tools for CrateDB developers, provides primitive functionality, to
```

Reply:
I didn't want to say "primitive functionality", but rather "primitive element" instead. Do you have a different suggestion?

-- https://www.techtarget.com/whatis/definition/primitive

Reply:
I've amended the patch once more, and used "primitive elements" literally now. Maybe this causes less hiccups for the uninitiated?

Reply:
it's minor detail, no need to change. Just, for me, it would read better as `basic functionality`, as this: `primitive elements` suit better in a programming language context.

Reply:
I see your point. In this case, the page about software testing probably has an audience of software developers. For me, it is difficult to designate something as `basic functionality`, when effectively it is `advanced functionality` already, wrapped into a primitive interface.

Now, that we talked about our personal dilemmas in wording extensively, let's merge the PR ;].